### ISSUE TITLE
Reading the RGB values

### ISSUE THREAD

**Issue by FernChaves** [darktable-org/darktable#12046]
Hi,
Congratulations for the excellent software.
It would be useful to be able to read the RGB values ​​when the cursor is on top of the image. 
For example, to know what are the values ​​of the lightest or darkest areas.
In other image editors these values ​​are displayed above or below the histogram as the cursor is moved over the image.
Thanks.

**Comment by gi-man** [comment]
It seems like you are asking about the color picker module in the darkroom view.  The manual describes it: 
https://docs.darktable.org/usermanual/3.8/en/module-reference/utility-modules/darkroom/global-color-picker/

**Comment by FernChaves** [comment]
Yes, that's it. I didn't find it before. I'm new to darktable. Thank you.

**Comment by gi-man** [comment]
No problem, can you close the issue?
